nautobot/ui/src/components/ObjectTable/ObjectListTable.js

```diff
@@ -113,7 +120,13 @@ export default function ObjectListTable({
     );
 
     const filtersPanel = useFiltersPanel({
-        content: <Text>You have successfully opened filters panel.</Text>,
+        content: (
+            <FiltersPanelContent
+                // TODO(norbert-mieczkowski-codilime): use actual lookup fields rather than table headers
```

Comment:
Do you need help from others on implementing the REST API to retrieve the lookup fields?

Reply:
Likely, I don't feel proficient in the Nautobot backend enough to do it myself

Reply:
Tracking as a new story